ingenialink/servo.py

```diff
@@ -381,6 +381,8 @@ def save_configuration(self, config_file, subnode=None):
         for subnode in subnodes:
             registers_dict = self.dictionary.registers(subnode=subnode)
             for reg_id, register in registers_dict.items():
+                if (register.address_type == REG_ADDRESS_TYPE.NVM_NONE) or (register.access != REG_ACCESS.RW):
```

Comment:
Is the second condition duplicated in line 390?

`if access_ops[register.access] == 'rw':`